eZ/Publish/Core/MVC/Symfony/Matcher/MatcherInterface.php

```diff
@@ -24,4 +26,6 @@
      * @throws \InvalidArgumentException Should be thrown if $matchingConfig is not valid.
      */
     public function setMatchingConfig($matchingConfig);
+
+    public function match(View $view);
```

Comment:
BC break: Think about custom matchers.
Best would probably be to introduce a new interface extending the old one.